### PR TITLE
LUN-351: Write Exercise incorrect answer validation 

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: release-notes/unreleased/LUN-351-separated-words-wrong.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: LUN-280
+show_in_stores: true
+platforms:
+  - android
+  - ios
+de: Richtige Antworten für getrennte Wörter werden nun nicht mehr als falsch angezeigt.
```
Comment (SteffiStuffel) [inline release-notes/unreleased/LUN-351-separated-words-wrong.yml:6]: (opional) werden nun korrekt ausgewertet.

File: src/routes/write-exercise/components/__tests__/InteractionSection.spec.tsx [hunk 2]
```diff
@@ -72,4 +72,68 @@ describe('InteractionSection', () => {
     fireEvent.press(button)
     await waitFor(() => expect(getByTestId('popover').props.isVisible).toBeTruthy())
   })
+
```
Comment (SteffiStuffel) [inline src/routes/write-exercise/components/__tests__/InteractionSection.spec.tsx:75]: Can you add a test case for the bug you fixed, so if the correct solution is built up of multiple words?

**Review by SteffiStuffel — APPROVED** [review]
Tested on Android emulator. "Der kontaktlose Spannungsprüfer" is correct now ;)

**Review by f1sh1918 — APPROVED** [review]
tested on ios. works fine

**Pull request merged**